framework/caching-cmm.md

```diff
@@ -149,3 +162,101 @@ the caching CMM MUST NOT store the decryption materials in the underlying CMC.
 ## Security Considerations
 
 [TODO: What security properties can the caching CMM guarantee?]
+
+## Appendix A: Cache Entry Identifier Formulas
+
+When accessing the underlying CMC,
+the caching CMM MUST use the formulas specified in this appendix
+in order to compute the cache entry identifier (also known as the cache key).
```

Comment:
The spec refers to a cache identifier
https://github.com/awslabs/aws-encryption-sdk-specification/blob/master/framework/cryptographic-materials-cache.md#cache-identifier
To be consistent we should use that and link to that definition.